lib/dynbuf.h

```diff
@@ -99,4 +99,6 @@ char *Curl_dyn_take(struct dynbuf *s, size_t *plen);
 #define DYN_MQTT_RECV       (64*1024)
 #define DYN_MQTT_SEND       0xFFFFFFF
 #define DYN_CRLFILE_SIZE    (400*1024*1024) /* 400mb */
+#define DYN_CERTFILE_SIZE   (100*1024) /* 1KiB */
+#define DYN_KEYFILE_SIZE    (100*1024) /* 1KiB */
```

Comment:
These limits seem pretty small to me, e.g. a PEM encoded [RSA 4096 client key](https://github.com/rustls/rustls/blob/main/test-ca/rsa-4096/client.key) is ~3.3kb

Since this is only being applied in `vtls/rustls.c` what about using something like the [`MAX_PAYLOAD`](https://github.com/rustls/rustls/blob/75ab55b9e44766c7af399c57436808d6961db265/rustls/src/msgs/message/mod.rs#L253C26-L253C39) size from Rustls allows:

```rust
/// Maximum message payload size.
/// That's 2^14 payload bytes and a 2KB allowance for ciphertext overheads.
const MAX_PAYLOAD: u16 = 16_384 + 2048;
```

A cert that can't be put into a TLS payload won't be of any use right?

Reply:
Oh ! I read this as 1KB not 100 KB.

It seems fine to me. Sorry!

Reply:
> Does that sound more reasonable?

Yes, for sure :-) Thanks!